tests/smoke.rs

```diff
@@ -4,6 +4,20 @@ use std::thread;
 static LIBBACKTRACE: bool = cfg!(feature = "libbacktrace") && !cfg!(target_os = "fuchsia");
 static GIMLI_SYMBOLIZE: bool = cfg!(all(feature = "gimli-symbolize", unix, target_os = "linux"));
 
+const HAVE_SP_IMPL: bool = cfg!(any(
```

Comment:
This looks pretty gnarly, I'm curious how you came across this? Ideally we just wouldn't list this at all here and we'd just always run the test?

Reply:
I took the `cfg`s from when dbghelp or libunwind are used and `any`'d them. I suppose I could remove this and just check that the returned SP is non-null, which would be easier to maintain.

Reply:
Done.